Dockerfile

```
@@ -5,6 +5,7 @@ FROM norsknettarkiv/jhove-warc-report-parser:0.1.0 as jwrp
 FROM python:3.9-alpine
 
 RUN apk add --no-cache jq
+RUN apk add --no-cache curl
```

```suggestion
RUN apk add --no-cache jq curl
```
As per https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers.